src/lib/gam.ts

```diff
@@ -1,9 +1,9 @@
 const CLICK_MACRO = '%%CLICK_URL_UNESC%%';
 const CACHE_BUST = '%%CACHEBUSTER%%';
 
-type GAMVariable<T extends string = string> = `[%${T}%]`;
+type GAMVariable<T extends string = string> = T;
```

Comment:
this is necessary so that the template parameters can be used for logic, as typescript complains about stuff like thsi otherwise.
```js
export let Var: GamVariable;

// ts complains because 'true' can never satisfy `[%${T}%]`
if (Var === 'true') {
//
}
```